modules/ipi-install-modifying-install-config-for-dual-stack-network.adoc

```diff
@@ -9,75 +9,87 @@ endif::[]
 
 :_mod-docs-content-type: PROCEDURE
 [id='modifying-install-config-for-dual-stack-network_{context}']
-= Deploying with dual-stack networking
+= Deploying static IP addressing with dual-stack networking
 
-For dual-stack networking in {product-title} clusters, you can configure IPv4 and IPv6 address endpoints for cluster nodes. To configure IPv4 and IPv6 address endpoints for cluster nodes, edit the `machineNetwork`, `clusterNetwork`, and `serviceNetwork` configuration settings in the `install-config.yaml` file. Each setting must have two CIDR entries each. For a cluster with the IPv4 family as the primary address family, specify the IPv4 setting first. For a cluster with the IPv6 family as the primary address family, specify the IPv6 setting first.
+Use the following procedure to deploy dual-stack static IP addressing for the bootstrap VM by using Ignition:
 
-[source,yaml]
+.Procedure
+
+. Create the Ignition configuration files:
++
+[source,terminal]
 ----
-machineNetwork:
-- cidr: {{ extcidrnet }}
-- cidr: {{ extcidrnet6 }}
-clusterNetwork:
-- cidr: 10.128.0.0/14
-  hostPrefix: 23
-- cidr: fd02::/48
-  hostPrefix: 64
-serviceNetwork:
-- 172.30.0.0/16
-- fd03::/112
+$ ./openshift-baremetal-install --dir <cluster_configs> create ignition-configs
 ----
++
+Replace `<cluster_configs>` with the path to your cluster configuration files.
 
-ifndef::vsphere[]
-[IMPORTANT]
-====
-On a bare-metal platform, if you specified an NMState configuration in the `networkConfig` section of your `install-config.yaml` file, add `interfaces.wait-ip: ipv4+ipv6` to the NMState YAML file to resolve an issue that prevents your cluster from deploying on a dual-stack network.
-
-.Example NMState YAML configuration file that includes the `wait-ip` parameter
-[source,yaml]
-----
-networkConfig:
-  nmstate:
-    interfaces:
-    - name: <interface_name>
-# ...
-      wait-ip: ipv4+ipv6
-# ...
+. Create the `bootstrap_config.sh` file:
++
+[source,terminal]
 ----
-====
-endif::vsphere[]
+#!/bin/bash
+
+BOOTSTRAP_CONFIG="[connection]
+type=ethernet
+interface-name=ens3
+[ethernet]
+[ipv4]
+method=manual
+addresses=<ipv4_address>/<cidr>
+gateway=<gateway_ipv4_address>
+dns=<dns_ipv4_address>"
+
+"[connection]
```

Comment:
This string starts out of blue and will be interpreted as a command. I think you should not close the string on line 41

Reply:
Updated as suggested.